rollup.config.js

```diff
@@ -3,41 +3,150 @@ import nodeResolve from 'rollup-plugin-node-resolve';
 import babel from 'rollup-plugin-babel';
 import replace from 'rollup-plugin-replace';
 import uglify from 'rollup-plugin-uglify';
+import pkg from './package.json';
 
-const env = process.env.NODE_ENV
-const config = {
-  output: {
-    format: 'umd',
-    name: 'ReduxPersist',
-    exports: 'named',
-    sourcemap: true,  
-  },
+// const ensureArray = maybeArr => Array.isArray(maybeArr) ? maybeArr : [maybeArr];
```

Comment:
this part is very much working, although not yet supporting `storage` and `.native.js` (they can be easily added, I just havent yet got to it)